paymentsheet/src/main/java/com/stripe/android/lpmfoundations/paymentmethod/PaymentMethodMetadata.kt

```diff
@@ -138,14 +141,31 @@ internal data class PaymentMethodMetadata(
             PaymentMethodRegistry.definitionsByCode[it]
         }.filter {
             it.isSupported(this)
-        }.filterNot {
+        }.withDefinitionIfSupported(
+            definition = LinkCardBrandDefinition,
+        ).filterNot {
```

Comment:
Link card brand is not included in the payment method types, so we need to add it after the fact.